torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1561,6 +1561,12 @@ def create_unbacked_symint(self):
         self.var_to_range[symbol] = ValueRanges(-sys.maxsize - 1, sys.maxsize)
         return SymInt(SymNode(symbol, self, int, None))
 
+    def create_unbacked_symbool(self):
+        symbol = sympy.Symbol(f"i{next(self.unbacked_symint_counter)}")
+        self.var_to_stack[symbol] = ''.join(traceback.format_list(traceback.extract_stack()[:-1]))
+        self.var_to_range[symbol] = ValueRanges.unknown()
+        return SymBool(SymNode(symbol, self, bool, None))
```

Comment:
This is awkward because the symbol is an int not a bool. Probably the easiest workaround is to define this as an unbacked symint and then test if it == 1 or something (value range is 0 to 1)